packages/adapter-node/src/handler.js

```diff
@@ -21,7 +21,8 @@ const protocol_header = env('PROTOCOL_HEADER', '').toLowerCase();
 const host_header = env('HOST_HEADER', 'host').toLowerCase();
 const body_size_limit = parseInt(env('BODY_SIZE_LIMIT', '524288'));
 
-const dir = fileURLToPath(import.meta.SERVER_DIR);
+const root = fileURLToPath(import.meta.SERVER_DIR);
```

Comment:
How come we can use `import.meta.SERVER_DIR` here but don't anywhere else?

Reply:
Unknown. This was like this before this PR. I believe it is defined in [/packages/adapter-node/index.js](/packages/adapter-node/index.js).

Reply:
ah, mb